package.json

```diff
@@ -6,7 +6,7 @@
     "build": "del-cli dist && tsc && copyfiles -u 3 \"src/utils/init-project/*\" dist/utils/init-project",
     "build:watch": "tsc -w",
     "build:grammar": "bash antlr/gradlew -p antlr generateGrammarSource",
-    "test": "jest --maxWorkers=4 --coverage",
+    "test": "jest --coverage",
```

Comment:
Oh, whoops. I forgot I did this.

I did it because one of the VS Code Jest extensions doesn't work correctly when `--maxWorkers` is set. Please run the tests locally and let me know if removing this has unsatisfactory results (for example, if it makes test runs a lot longer).

Reply:
The tests do take longer for me than they do on `master`, but I ran them a few times and I couldn't get them to be consistently shorter with `--maxWorkers=4` back. I think 3 out of 4 times using `--maxWorkers=4` was shorter, but that fourth time when it didn't make a difference makes me feel like it is probably fine to keep this as is. It could easily just be my computer.

Reply:
OK. We'll leave it for now. We can always bring it back later if it ends up being a problem.